src/TodoForm.jsx

```diff
@@ -0,0 +1,10 @@
+function TodoForm() {
+  return (
+    <>
+      <label htmlFor="todoTitle">Todo</label>
+      <input id="todoTitle" />
+      <button>Add Todo</button>
+    </>
+  );
```

Comment:
This would be better semantically (and later functionally) wrapped in a `form` tag, as you're inserting form elements here.

```suggestion
    <form>
      <label htmlFor="todoTitle">Todo</label>
      <input id="todoTitle" />
      <button>Add Todo</button>
    </form>
  );
```

Reply:
Thank you, I will revise that in my next PR